features/0348-transition-msg-type-to-https/README.md

```diff
@@ -11,15 +11,16 @@
 ## Summary
 
 Per issue [#225](https://github.com/hyperledger/aries-rfcs/issues/225), the
-Aries community has agreed to change the prefix for protocol message types from
-`did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/` to `https://didcomm.org/`. Examples of the two message types forms are:
+Aries community has agreed to change the prefix for protocol message types that currently use the
```

Comment:
I also think newly written RFCs should also use the new format. i.e., we shouldn't need to transition new efforts.